Sources/Brave/Frontend/Browser/Toolbars/UrlBar/CollapsedURLBarView.swift

```diff
@@ -49,20 +46,44 @@ class CollapsedURLBarView: UIView {
   }
   
   private func updateLockImageView() {
-    lockImageView.isHidden = false
+    secureContentStateView.isHidden = !secureContentState.shouldDisplayWarning
+    separatorLine.isHidden = secureContentStateView.isHidden
+    secureContentStateView.configuration = secureContentStateButtonConfiguration
+  }
+  
+  private var secureContentStateButtonConfiguration: UIButton.Configuration {
+    let clampedTraitCollection = traitCollection.clampingSizeCategory(maximum: .accessibilityLarge)
+    var configuration = UIButton.Configuration.plain()
+    configuration.preferredSymbolConfigurationForImage = .init(font: .preferredFont(forTextStyle: .caption1, compatibleWith: clampedTraitCollection), scale: .small)
+    configuration.buttonSize = .small
+    configuration.imagePadding = 4
+    configuration.contentInsets = .init(top: 0, leading: 0, bottom: 0, trailing: 0)
+    
+    var title = AttributedString(Strings.tabToolbarNotSecureTitle)
+    title.font = .preferredFont(forTextStyle: .caption1, compatibleWith: clampedTraitCollection)
+    
+    let isTitleVisible = !traitCollection.preferredContentSizeCategory.isAccessibilityCategory
     
     switch secureContentState {
-    case .localHost:
-      lockImageView.isHidden = true
-    case .insecure:
-      lockImageView.setImage(UIImage(braveSystemNamed: "leo.info.filled")?
-        .withRenderingMode(.alwaysOriginal)
-        .withTintColor(.braveErrorLabel), for: .normal)
-      lockImageView.accessibilityLabel = Strings.tabToolbarWarningImageAccessibilityLabel
-    case .secure, .unknown:
-      lockImageView.setImage(UIImage(braveSystemNamed: "brave.lock.alt", compatibleWith: nil), for: .normal)
-      lockImageView.accessibilityLabel = Strings.tabToolbarLockImageAccessibilityLabel
+    case .unknown, .localhost, .secure:
```

Comment:
what is .unknown? just wondering if we should show it to be the same as secure.

Reply:
if it could possibly be any kind of remote connection that isn't TLS-protected, we should show it as one of the insecure state.

Reply:
Unknown is just the default enum case thats set before the page has been evaluated, theoretically it should never actually be the value of the tab secure state for a loaded page

Reply:
on desktop it actually shows a circle warning icon in the urlbar briefly while the page is loading. can we do the same on iOS? this ensures that if something goes wrong in the load, the page doesn't end up showing up as secure.

Reply:
this also happens on Chrome iOS. ex: enter nonexistent.azuki.vip in the URL bar and note the warning info icon while the page is loading.

Reply:
Sure we can do that, I'll throw it together tomorrow